app/controllers/api/v0x1/portfolio_items_controller.rb

```diff
@@ -18,6 +18,13 @@ def create
         render :json => { :errors => e.message }, :status => :not_found
       end
 
+      def update
+        portfolio_item = PortfolioItem.find(params.require(:id))
+        portfolio_item.update!(params.permit(:favorite, :name, :description, :orphan, :state, :display_name, :long_description, :distributor, :documentation_url, :support_url))
```

Comment:
@lindgrenj6 - The permit method could get moved into a private params_permit method to hold with our current pattern for strong params.
I believe we 'may' need a make sure that any exception on the update is handled correctly when the `render` method is called, and is in line with openapi.json spec `401`, `403`, and `404`.